processor/src/main/java/org/mapstruct/ap/internal/model/common/HasFormattingParameters.java

```diff
@@ -0,0 +1,61 @@
+/**
+ *  Copyright 2012-2017 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.internal.model.common;
+
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.AnnotationValue;
+import javax.lang.model.element.Element;
+
+/**
+ * Formatting parameters for the common subpackage.
+ *
+ * @author Filip Hrisafov
+ */
+public interface HasFormattingParameters {
```

Comment:
I don't quite get why we need this interface... Do we need one in the common package? Could we also just move `FormattingParameters` to the common package? My main concern is that `HasFormattingParameters` reads a bit funny and I guess it's hard to come up with a better name... 🙄